### PR TITLE
[UK] Add `Staff Log` and `Staff Log 2007`

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by cl8n** [comment]
(in case you were planning to) it's probably not worth the time to translate the rest of these, they get very formulaic but with very subtle inconsistencies that have not yet been addressed -- one day I can probably have it auto-translate in the same manner as group pages but yeah. your call in the end but this is probably just busy work.

also it's not necessary to copy over or translate comments when they are about the content itself and not the writing/formatting/etc, they are just notes for future EN writers and don't affect the page at all.

File: wiki/People/Staff_log/2007/uk.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Історія команд osu! за 2007 рік
+
+*Примітка: Історія деяких користувачів може бути неповною протягом цього року. "Модератори" пізніше були перейменовані в "Beatmap Approval Team".*
+
+- [phill](https://osu.ppy.sh/users/53) додали до модераторів <!-- set up website and IRC? https://osu.ppy.sh/community/forums/topics/21 -->
```
Comment (JyJyJ4JyJ) [inline wiki/People/Staff_log/2007/uk.md:5]: мб "доданий" краще звучатиме

File: wiki/People/Staff_log/uk.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+tags:
+  - promotion
+  - retirement
+  - просування
+  - відставка
+---
+
+# Історія команд osu!
+
+Тут перелічені зміни у складах команд osu! за різні роки, щоб їх було простіше відстежувати.
```
Comment (JyJyJ4JyJ) [inline wiki/People/Staff_log/uk.md:11]:
```suggestion
Тут перелічені зміни у складі команди osu! за різні роки, щоб їх було простіше відстежувати.
```

File: wiki/People/Staff_log/uk.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+tags:
+  - promotion
+  - retirement
+  - просування
+  - відставка
+---
+
+# Історія команд osu!
```
Comment (JyJyJ4JyJ) [inline wiki/People/Staff_log/uk.md:9]:
```suggestion
# Історія команди osu!
```

**Review by cl8n — APPROVED** [review]

**Pull request merged**